web/src/main/java/com/google/sps/servlets/SaveStockServlet.java

```diff
@@ -47,6 +49,41 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     long timestamp = System.currentTimeMillis();
 
+    Date date = Calendar.getInstance().getTime();
+    String gooddate = date.toString();
+    String[] date_ok = gooddate.split(" ");
+
+    if (date_ok[0].equalsIgnoreCase("Sun")) {
+      date_ok[0] = "Sunday";
+    } else if (date_ok[0].equalsIgnoreCase("Mon")) {
+      date_ok[0] = "Monday";
+    } else if (date_ok[0].equalsIgnoreCase("Tue")) {
+      date_ok[0] = "Tuesday";
+    } else if (date_ok[0].equalsIgnoreCase("Wed")) {
+      date_ok[0] = "Wednesday";
+    } else if (date_ok[0].equalsIgnoreCase("Thu")) {
+      date_ok[0] = "Thursday";
+    } else if (date_ok[0].equalsIgnoreCase("Fri")) {
+      date_ok[0] = "Friday";
+    } else {
+      date_ok[0] = "Saturday";
+    }
+    if (date_ok[1].equalsIgnoreCase("Mar")) {
+      date_ok[1] = "March";
```

Comment:
Why is March the only one here, what about the other dates of the year?

Reply:
I will add them now